tests/app/template/test_rest.py

```diff
@@ -37,13 +37,13 @@
 from tests.conftest import set_config_values
 
 
-@pytest.mark.xfail(reason='Mislabelled for route removal, fails when unskipped', run=False)
+@pytest.mark.skip(reason='TODO #2336 - Fail due to orphaned User object.')
+@pytest.mark.serial
```

Comment:
I don't think this needs to be serial.  There's no GET request to a list endpoint.

Reply:
Updated.